ERCS/erc-7752.md

```diff
@@ -0,0 +1,607 @@
+---
+eip: 7752
+title: Equity Token
+description: Equity token for corporations
+author: Matt Rosendin (@mrosendin) <matt@capsign.com>
+discussions-to: https://ethereum-magicians.org/t/erc-7752-equity-token/20735
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-08-06
+requires: 173
+---
+
+## Abstract
+
+An ERC token standard representing shares in a corporation. This new interface standardizes equity management for issuers, investors, transfer agents, and financial intermediaries.
+
+## Motivation
+
+Creating a standard for corporation requires an industry effort. That's why this token standard adheres to the same design principles as the Open Cap Format (OCF), an industry-approved data standard for cap tables. Now everyone can adopt the same OCF-compliant token standard for Ethereum and beyond!
+
+## Specification
+
+> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+MUST NOT use [ERC-20](./eip-20.md) methods, which have breaking changes.
+
+SHOULD use `addShareClass(bytes32 name, string uri)` to create at least one share class before minting tokens.
+
+SHOULD use `balanceOf(address user)` to get a holder's total share count, incuding all share classes.
+
+SHOULD use `balanceOf(address user, string name)` to lookup a holder's balance by the specified share class name.
+
+SHOULD use `totalSupply()` to get the total outstanding share count, incuding all share classes.
+
+SHOULD use `totalSupply(string name)` to lookup outstanding shares by the specified class name.
+
+SHOULD use `securities(bytes32 securityId)` to fetch individual security details.
+
+MAY assign a `uri` when creating a share class that points to a JSON file containing relevant metadata.
+
+MAY specify a value of `0` for `startTime`, `cliffTime`, and `duration` parameters in `issue` for stock issuances without a vesting condition.
```

Comment:
It is unclear if these are requirements on the implementation or on consumers of the interface.